CONTRIBUTING.md

```diff
@@ -262,6 +272,13 @@ The MISRA standard is more exacting that the OSCC standard. It is not implied th
 
 5. Structure, union, and enumeration tags must be unique
 
+6. Variable, macro and function naming should use “snake case”

```

Comment:
Can we enforce that names are also human readable for both variables and macros?

For example `PEDAL_THRESH`would be rejected for `PEDAL_THRESHOLD`.